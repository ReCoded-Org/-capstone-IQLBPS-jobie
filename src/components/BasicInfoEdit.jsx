/* eslint-disable jsx-a11y/label-has-associated-control */
import React from "react";

function BasicInfoEdit() {
  return (
    <div className="bg-gray-50 pt-1 pb-0">
      <h1 className="text-2xl text-left ml-48 mt-24 small:ml-16 medium:ml-24">
        Basic Info
      </h1>

      <form action="#" method="POST">
        <div className=" flex justify-center	pb-20 	 overflow-hidden  sm:rounded-md">
          <div className="w-3/4		 px-4 py-5 sm:p-6">
            <div className="grid grid-cols-9 gap-6">
              <div className="col-span-6 sm:col-span-3">
                <label
                  htmlFor="first-name"
                  className="block text-sm font-medium text-gray-700"
                >
                  First Name
                </label>
                <input
                  placeholder="First name"
                  type="text"
                  name="first-name"
                  id="first-name"
                  autoComplete="given-name"
                  className="h-9 mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
                />
              </div>

              <div className="col-span-6 sm:col-span-3">
                <label
                  htmlFor="last-name"
                  className="block text-sm font-medium text-gray-700"
                >
                  Last Name
                </label>
                <input
                  placeholder="Last name"
                  type="text"
                  name="last-name"
                  id="last-name"
                  autoComplete="family-name"
                  className="h-9 mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
                />
              </div>

              <div className="col-span-6 sm:col-span-3">
                <label
                  htmlFor="Title"
                  className="block text-sm font-medium text-gray-700"
                >
                  Title
                </label>
                <input
                  type="text"
                  name="Title"
                  id="Title"
                  autoComplete="given-name"
                  className="h-9 mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
                />
              </div>

              <div className="col-span-6 sm:col-span-9 ">
                <label
                  htmlFor="Biography"
                  className="block text-left text-sm font-medium text-gray-700"
                >
                  Biography
                </label>
                <textarea
                  type="text"
                  name="Biography"
                  id="Biography"
                  className=" h-24 w-full mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
                />
              </div>
            </div>
          </div>
        </div>
      </form>
    </div>
  );
}

export default BasicInfoEdit;
