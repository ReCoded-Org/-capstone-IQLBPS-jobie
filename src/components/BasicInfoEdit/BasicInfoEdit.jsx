/* eslint-disable jsx-a11y/label-has-associated-control */
import React from "react";
import {  useTranslation } from "react-i18next";

function BasicInfoEdit() {
  const {t}= useTranslation();
  return (
    <div className="bg-gray-50 pt-1 pb-0">
      <h1 className="text-2xl text-left ml-48 mt-24 small:ml-16 medium:ml-24">
        {t('basic-info')}
      </h1>

      <form action="#" method="POST">
        <div className=" flex justify-center	pb-20 	 overflow-hidden  sm:rounded-md">
          <div className="w-3/4		 px-4 py-5 sm:p-6">
            <div className="grid grid-cols-9 gap-6">
              <div className="col-span-6 sm:col-span-3">
                <label
                  htmlFor="first-name"
                  className="block text-sm font-medium text-gray-700"
                >
                  {t('first-Name')}
                </label>
                <input
                  placeholder="First name"
                  type="text"
                  name="first-name"
                  id="first-name"
                  autoComplete="given-name"
                  className="h-9 mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
                />
              </div>

              <div className="col-span-6 sm:col-span-3">
                <label
                  htmlFor="last-name"
                  className="block text-sm font-medium text-gray-700"
                >
                  {t('last-name')}
                </label>
                <input
                  placeholder="Last name"
                  type="text"
                  name="last-name"
                  id="last-name"
                  autoComplete="family-name"
                  className="h-9 mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
                />
              </div>

              <div className="col-span-6 sm:col-span-3">
                <label
                  htmlFor="Title"
                  className="block text-sm font-medium text-gray-700"
                >
                  {t('title')}
                </label>
                <input
                  type="text"
                  name="Title"
                  id="Title"
                  autoComplete="given-name"
                  className="h-9 mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
                />
              </div>

              <div className="col-span-6 sm:col-span-9 ">
                <label
                  htmlFor="Biography"
                  className="block text-left text-sm font-medium text-gray-700"
                >
                  
                
                
                {t('biography')}
                </label>
                <textarea
                  type="text"
                  name="Biography"
                  id="Biography"
                  className=" h-24 w-full mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
                />
              </div>
            </div>
          </div>
        </div>
      </form>
    </div>
  );
}

export default BasicInfoEdit;
