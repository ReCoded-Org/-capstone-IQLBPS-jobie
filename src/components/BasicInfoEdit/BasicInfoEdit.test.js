import renderer from "react-test-renderer";
import BasicInfoEdit from "./BasicInfoEdit";

it("renders correctly when the component matches the snapshot", () => {
  const tree = renderer.create(<BasicInfoEdit />).toJSON();
  expect(tree).toMatchSnapshot();
});
