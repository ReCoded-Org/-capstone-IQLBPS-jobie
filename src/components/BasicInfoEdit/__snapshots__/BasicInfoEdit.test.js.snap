// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when the component matches the snapshot 1`] = `
<div
  className="bg-gray-50 pt-1 pb-0"
>
  <h1
    className="text-2xl text-left ml-48 mt-24 small:ml-16 medium:ml-24"
  >
    basic-info
  </h1>
  <form
    action="#"
    method="POST"
  >
    <div
      className=" flex justify-center	pb-20 	 overflow-hidden  sm:rounded-md"
    >
      <div
        className="w-3/4		 px-4 py-5 sm:p-6"
      >
        <div
          className="grid grid-cols-9 gap-6"
        >
          <div
            className="col-span-6 sm:col-span-3"
          >
            <label
              className="block text-sm font-medium text-gray-700"
              htmlFor="first-name"
            >
              first-Name
            </label>
            <input
              autoComplete="given-name"
              className="h-9 mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
              id="first-name"
              name="first-name"
              placeholder="First name"
              type="text"
            />
          </div>
          <div
            className="col-span-6 sm:col-span-3"
          >
            <label
              className="block text-sm font-medium text-gray-700"
              htmlFor="last-name"
            >
              last-name
            </label>
            <input
              autoComplete="family-name"
              className="h-9 mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
              id="last-name"
              name="last-name"
              placeholder="Last name"
              type="text"
            />
          </div>
          <div
            className="col-span-6 sm:col-span-3"
          >
            <label
              className="block text-sm font-medium text-gray-700"
              htmlFor="Title"
            >
              title
            </label>
            <input
              autoComplete="given-name"
              className="h-9 mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
              id="Title"
              name="Title"
              type="text"
            />
          </div>
          <div
            className="col-span-6 sm:col-span-9 "
          >
            <label
              className="block text-left text-sm font-medium text-gray-700"
              htmlFor="Biography"
            >
              biography
            </label>
            <textarea
              className=" h-24 w-full mt-1 block rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
              id="Biography"
              name="Biography"
              type="text"
            />
          </div>
        </div>
      </div>
    </div>
  </form>
</div>
`;
